.github/workflows/main.yml

```diff
@@ -165,12 +165,41 @@ jobs:
       #       'halt\n' \
       #     | docker exec -i $CONTAINER iris session iris -UUSER
       #     docker stop $CONTAINER
+      - name: Clean install v0.7.x and test migration to latest.
+        timeout-minutes: 15
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl http://localhost:52773/registry/packages/-/all | jq
+          curl http://localhost:52773/registry/packages/zpm/ | jq
+          ASSET_NAME='zpm-0.7.1-beta.4.xml'
+          ASSET_URL=`wget --header "Authorization: token ${GITHUB_TOKEN}" -qO- https://api.github.com/repos/intersystems/ipm/releases | jq -r ".[].assets[] | select(.name == \"${ASSET_NAME}\") | .browser_download_url"`
+          wget $ASSET_URL -O /tmp/zpm.xml
+          docker run --network zpm --rm -d ${{ steps.image.outputs.name }} ${{ steps.image.outputs.flags }}
+          CONTAINER=`docker ps -lq`
```

Comment:
Alternatively, can also use `CONTAINER=$(docker run xxx)`. This way, even if the order of lines is changed, or a new `docker run` command is inserted between the above `docker run` and `docker ps -lq`, we still get a hash of the correct container. If you want to log the container ID for debugging purposes, you can run an `echo $CONTAINER` afterwards.